eng/config/PublishData.json

```diff
@@ -78,7 +80,9 @@
       "Microsoft.CodeAnalysis.VisualBasic.Features": "arcade",
       "Microsoft.CodeAnalysis.VisualBasic.Workspaces": "arcade",
       "Microsoft.CodeAnalysis.Build.Tasks": "arcade",
+      "Microsoft.Net.Compilers": "arcade",
```

Comment:
should we make a separate package feed that contains the older packages?  Which would make it more obvious that we can delete them when we delete the servicing branch config too